household_info.csv

```diff
@@ -0,0 +1,13 @@
+country_code_12,lower_super_output_area_code_11,postcode,unique_property_reference_code,town_name,ctry18nm,la_code,ew,address_type,council_tax,udprn,address_base_postal,LSOA11NM,cis_area_code_20,RGN19CD,LAD20CD,LAD20NM,country_name_12,number_of_households_population_by_cis,number_of_households_population_by_country
```

Comment:
Two csv files have been included here - please write these outside of the repo or avoid adding them to commits